0_getting_started.ipynb

```diff
@@ -0,0 +1,195 @@
+{
```

Comment:
<p>Add links to each section.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pyvista/pyvista-tutorial/pull/9/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>